socialdistribution_root/socialdistribution/utils.py

```diff
@@ -12,4 +12,15 @@ def getAcceptedMediaType(request: HttpRequest):
 
     @staticmethod
     def serialize(serializer: ModelSerializer, request: HttpRequest):
-        return JSONRenderer().render(serializer.data, Utils.getAcceptedMediaType(request))
\ No newline at end of file
+        return JSONRenderer().render(serializer.data, Utils.getAcceptedMediaType(request))
+
+    
+    # Used for formatting and styling responses
+    @staticmethod
+    def compose_posts_dict(query_type, data):
```

Comment:
we should probably give it a better name at some point if it is being used for all kind of stuff 😅 
Maybe like, compose response, or format response

Reply:
Good point, I'll do that